FAQs/README.md

```diff
@@ -338,3 +341,50 @@ stages:
+
+3. In case access is recently granted, it will reflect in AzTS from next scan onwards. So in case of automated AzTS scan it would take next 24 hours to reflect latest access. You can use the [On-Demand scan](README.md#2-manually-trigger-azts-on-demand-scan-for-entire-tenant) command to trigger the scan in case you want to reflect newly added access immediately.
+
+### **TroubleShooting Steps:**
+
+Please check if below required permissions are granted. These permissions require admin consent. Therefore, user must be a member of one of the following administrator roles: </br>Global Administrator or Privileged Role Administrator.
+
+**1. 'User.Read' permission for AzTS UI AD application**
+
+This permission is used to read logged in user's details such as name, email, and photo.  
+
+> **Note:** If you do not have the permission to grant this permission, please contact your administrator to complete the setup using [this PowerShell script](../Scripts/ScriptToSetupAzureADApplicationForAzTSUI.ps1?raw=1). To run this script, you need to provide the subscription id and resource group name in which AzTS solution needs to be installed.
+
+
+**2. MS Graph "PrivilegedAccess.Read.AzureResources" and "Directory.Read.All" permissions for Scanner identity (MI)** 
+ 
+This permission is used to read data in your organization's directory, such as users, groups and apps and to validate Role-based access control (RBAC) using Azure AD Privileged Identity Management (PIM). 
+</br>If you do not have the required permission, please contact your administrator to get "PrivilegedAccess.Read.AzureResources" and "Directory.Read.All" permission for your scanner MI in Azure Active Directory using [this PowerShell script](../Scripts/ScriptToGrantGraphPermissionToScannerMI.ps1?raw=1). To run this script, you need to provide the object id of the user-assigned managed identity (scanner MI) which will be available in deployment log file.
+
+In case you are using Privileged Identity Management(PIM) for assigning access, please make sure that below settings are enabled:
+1. Go to Azure Portal.
+2. Navigate to AzTS host subscription -> AzTS host resource group.
+3. Go to AzTS Metadata Aggregator function app (AzSK-AzTS-MetadataAggregator-xxxxx).
+4. Go to Configurations under Settings.
+5. Check if setting AuthzSettings__IsPIMEnabled is available. If not, add new application setting 'AuthzSettings__IsPIMEnabled' with value as true. Click 'OK'. (This will enable AzTS to fetch role assignemnts using PIM API calls).
+6. Click 'Save'.
+7. Also check whether MS based graph access are granted to Scanner MI as mentioned above. If these permissions are granted post AzTS setup completion, then re-run AzTS setup installation command (Install-AzSKTenantSecuritySolution) with parameter -ScanIdentityHasGraphPermission:$true. More details can be found [here](../01-Setup%20and%20getting%20started/README.md#step-6-of-6-run-setup-command)
```

Comment:
Additionally, please verify if the necessary Microsoft Graph permissions have been granted to the Scanner Managed Identity (MI), as mentioned earlier. If these permissions are granted after the completion of AzTS setup, you will need to rerun the AzTS setup installation command (Install-AzSKTenantSecuritySolution) with the parameter -ScanIdentityHasGraphPermission set to $true. For more detailed information, you can refer here.

Reply:
Updated